jetty-client/src/main/java/org/eclipse/jetty/client/AbstractConnectionPool.java

```diff
@@ -321,7 +323,7 @@ protected Connection activate()
                     EntryHolder holder = (EntryHolder)((Attachable)connection).getAttachment();
                     if (holder.isExpired(maxDurationNanos))
                     {
-                        boolean canClose = remove(connection, true);
+                        boolean canClose = remove(connection);
```

Comment:
Deprecate `remove(Connection, boolean)` and move its implementation in `remove(Connection)`.

Reply:
Done.